docs/nixos-with-flakes/other-useful-tips.md

```diff
@@ -21,7 +21,7 @@ This way, you can use Git to manage the configuration in `~/nixos-config`. The c
 Another approach is to delete `/etc/nixos` directly and specify the configuration file path each time you deploy it:
 
 ```shell
```

Comment:
The purpose here is to delete `/etc/nixos`, we do not need it in this approach, so `mv` should still be used here.